contracts/utils/contracts/src/MixinReentrancyGuardRichErrors.sol

```diff
@@ -0,0 +1,46 @@
+/*
+
+  Copyright 2019 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.5.9;
+
+import "./RichErrors.sol";
+
+
+contract MixinReentrancyGuardRichErrors is
+    RichErrors
+{
+    // bytes4(keccak256("IllegalReentrancyError(uint256,uint256)"))
+    bytes4 internal constant ILLEGAL_REENTRANCY_ERROR_SELECTOR =
+        0x8db7de08;
+
+    // solhint-disable func-name-mixedcase
+    function IllegalReentrancyError(
+        uint256 globalReentrancyCounter,
+        uint256 localReentrancyCounter
+    )
+        internal
+        pure
+        returns (bytes memory)
+    {
+        return abi.encodeWithSelector(
+            ILLEGAL_REENTRANCY_ERROR_SELECTOR,
+            globalReentrancyCounter,
```

Comment:
I feel like the counters aren't really useful here, we can just return the hard coded byte array that is equal to `abi.encodePacked(ILLEGAL_REENTRANCY_ERROR_SELECTOR)` (note the selector will change when we remove these params).